cmd/podman/top.go

```diff
@@ -79,3 +84,48 @@ func topCmd(c *cli.Context) error {
 	}
 	return nil
 }
+
+func filterPids(psOutput string, pids []string) ([]string, error) {
```

Comment:
This feels like it should happen in libpod, before we return from `GetContainerPidInformation` - I don't think we'll ever want to call that and get PIDs not in the container?

Reply:
And here I was thinking you would say the **exact** opposite.

Reply:
fixed